solr/contrib/ltr/src/java/org/apache/solr/ltr/util/CommonLTRParams.java

```diff
@@ -1,34 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.solr.ltr.util;
-
-public class CommonLTRParams {
-
-  /** name of the attribute containing the feature name **/
-  public static final String FEATURE_NAME = "name";
-  /** name of the attribute containing the normalizer **/
-  public static final String FEATURE_NORM = "norm";
-  /** name of the attribute containing the model name **/
-  public static final Object MODEL_NAME = "name";
-  /** name of the attribute containing the model type **/
-  public static final Object MODEL_CLASS = "class";
-  /** name of the attribute containing the feature store used **/
-  public static final Object MODEL_FEATURE_STORE = "store";
```

Comment:
hmmm I'm not sure if moving this out of a publically accessible location into private variables is completely ideal. Aren't we cutting down on extensibility a bit?